### ISSUE TITLE
Static content deploy produces strange folder names

### ISSUE THREAD

**Issue by friimaind** [magento/magento2#14271]
### Preconditions
Magento 2.1.9
PHP 7.0.27
Apache/2.4.6 (CentOS)

### Steps to reproduce
rm -rf pub/static/*
php bin/magento setup:static-content:deploy

### Expected result
static content to be deployed

### Actual result
My static content have strange folder names:

> )
)))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))
�''�""
�'�"
1'"
${10000294 9999730}
${10000417 10000482}
1BpM85pF'));select pg_sleep(3); -- 
-1' OR 2 25-25-1=0 0 0 1 or 'm91vQSPl'='
-1" OR 2 271-271-1=0 0 0 1 -- 
-1 OR 2 294-294-1=0 0 0 1 -- 
-1 OR 2 301-301-1=0 0 0 1 -- 
-1' OR 2 302-302-1=0 0 0 1 or 'DTrsB2ld'='
-1' OR 2 408-408-1=0 0 0 1 or 'qzexiEUS'='
-1' OR 2 469-469-1=0 0 0 1 -- 
-1" OR 2 537-537-1=0 0 0 1 -- 
-1 OR 2 615-615-1=0 0 0 1
-1 OR 2 661-661-1=0 0 0 1
-1' OR 2 75-75-1=0 0 0 1 -- 
-1 OR 2 813-813-1=0 0 0 1 -- 
-1" OR 2 826-826-1=0 0 0 1 -- 
-1 OR 2 871-871-1=0 0 0 1
-1' OR 2 896-896-1=0 0 0 1 -- 
1 waitfor delay '0:0:10' -- 
1 waitfor delay '0:0:12' -- 
1 waitfor delay '0:0:14' -- 
1 waitfor delay '0:0:6' -- 
${9999010 10000387}
${9999246 9999551}
${9999933 9999712}
@@9k8ix
adminhtml
azg4WjRqZG0=
Acme
bqxQWM8L');select pg_sleep(6); -- 
bWZZSGN6aW0=
CgttiUE5');select pg_sleep(10); -- 
CVr4um7H'));select pg_sleep(15); -- 
deployed_version.txt
dFpEenZOQks=
elJhFmMN';select pg_sleep(18); -- 
em46vvjT'; waitfor delay '0:0:9' -- 
@@eNcv4
@@eVpMX
frontend
frontend' AND 2*3*8=6*8 AND 'CvtC'='CvtC
frontend%' AND 2*3*8=6*8 AND 'G5C3'!='G5C3%
frontend' AND 2*3*8=6*8 AND 'MH8i'='MH8i
frontend" AND 2*3*8=6*8 AND "U4Ga"="U4Ga
frontend" AND 2*3*8=6*8 AND "U6iP"="U6iP
frontend%' AND 2*3*8=6*8 AND 'wt85'!='wt85%
FZOee6vX');select pg_sleep(12); -- 
g91qA8ob'));select pg_sleep(12); -- 
jE4VVXhs'));select pg_sleep(7); -- 
JyI=
K2cgJ3xZ';select pg_sleep(10); -- 
ktILQAfe'; waitfor delay '0:0:10' -- 
kYBVILly
$(nslookup DubX4Dx7)
$(nslookup eBbaX0RE)
$(nslookup N1eiAsK2)
$(nslookup qo4MZwT7)
$(nslookup tI4GQ4ep)
o4M1doD2'; waitfor delay '0:0:21' -- 
@@PO5lY
;print(md5(acunetix_wvs_security_test));
${@print(md5(acunetix_wvs_security_test))}
';print(md5(acunetix_wvs_security_test));$a='
";print(md5(acunetix_wvs_security_test));$a="
pRrfJsMw';select pg_sleep(9); -- 
Q3pORGVqb2M=
qMHyWy81
_requirejs
RL6EMrX7'; waitfor delay '0:0:12' -- 
@@sAWjM
(select convert(int,CHAR(65)))
testasp.vulnweb.com
uYONDqnF';select pg_sleep(12); -- 
VpjeLXh5
vWOzqQnS');select pg_sleep(3); -- 
VxGdqDaS
WEB-INF
WjgP1RFh'; waitfor delay '0:0:12' -- 
wQdT9j47';select pg_sleep(7); -- 
yIFIqWj1'));select pg_sleep(6); -- 
YThOVlhLWXM=
z3RqXfGG');select pg_sleep(7); -- 
Zh6ziRnY

**Comment by magento-engcom-team** [comment]
@friimaind thank you for your report.
We were not able to reproduce this issue by following the steps you provided. Please provide more detailed steps to reproduce or try to reproduce this issue on a clean installation or latest release.

**Comment by friimaind** [comment]
Found the problem: it was my mistake.  A big one.

I have two same installations: "stage" and "production". I was aligning stage with production copying files and database. I launched php bin/magento setup:static-content:deploy from "stage" **without** changing the env.php (which was pointing to "production" installation).

This mistake created corrupted symlinks into production static folder (files/folders + symlink pointing to stage files/folders).
After a moment of panic i deleted all static content, restored /lib/ folder which, moreover, was corrupted (missing folder, files) and re-launched static-content:deploy with success.
